lib/inc/cpp-pcp-client/protocol/errors.hpp

```diff
@@ -0,0 +1,12 @@
+/**
+ * This is a shim to maintain compatibility with pxp-agent during development.
+ */
+#pragma once
+
+#include <cpp-pcp-client/protocol/v1/errors.hpp>
+
+namespace PCPClient {
+
+using namespace v1;
```

Comment:
Is this "exported", i.e. does this really make symbols from the `PCPClient::v1` namespace appear in the `PCPClient` namespace for the consumers of this header?
I haven't tested it, I'm just trying to make sense of the definition given [here](http://en.cppreference.com/w/cpp/language/namespace) in point 5. The 
> until the end of the scope in which it appears
clause makes me suspicious.